tokens/blocket.se/badge.yml

```diff
@@ -4,26 +4,26 @@ token: maps
 color:
   badge:
     neutral:
-      text: bluegray-900
+      text: s-color-text
       background: gray-100
     info:
-      text: aqua-900
-      background: aqua-100
+      text: s-color-text
+      background: s-color-background-info-subtle
     positive:
-      text: green-900
-      background: green-100
+      text: s-color-text
+      background: s-color-background-positive-subtle
     warning:
-      text: yellow-900
-      background: yellow-100
+      text: s-color-text
+      background: s-color-background-warning-subtle
     negative:
-      text: red-900
-      background: red-100
+      text: s-color-text
+      background: s-color-background-negative-subtle
     disabled:
-      text: bluegray-900
-      background: bluegray-300
+      text: s-color-text
+      background: s-color-background-disabled
     price:
-      text: white
-      background: gray-900-transparentized-70-percent
+      text: s-color-text-inverted
+      background: gray-900-alpha70
```

Comment:
Should this be `s-color-background-inverted`?

Reply:
Correct. Blocket should have the "default" semantic tokens and no overrides at this point. 👍 🙏  I fix. 🙂

Reply:
I don't understand why we want to have a different setup for blocket 🤔 Can't all brands use a `s-color-background-inverted` that has a value of `gray-900-alpha70`?

Reply:
Ah, now I see that there already exists a `s-color-background-inverted` token with `gray-900` value and that `gray-900-alpha70` is an "override" on the badge component for Finn and Tori.

Reply:
Correct again. 😄 😜